docker_templates/templates/docker_images_ros2/nightly/create_ros_image.Dockerfile.em

```diff
@@ -20,6 +20,7 @@
 ))@
 @{
 template_dependencies = [
+    'cmake',
```

Comment:
`git` should be added as a template dependency as well

Reply:
Added `git` dependency in f99dd82